components/cam/cime_config/config_component.xml

```diff
@@ -427,6 +430,7 @@
     <value compset="DOCN%AQP">$SRCROOT/components/cam/cime_config/usermods_dirs/aquap</value>
     <value compset="DOCN%SOMAQP">$SRCROOT/components/cam/cime_config/usermods_dirs/aquap</value>
     <value compset="CAM5%RCE">$SRCROOT/components/cam/cime_config/usermods_dirs/rcemip</value>
+    <value compset="CAM5%MMF1-RCE">$SRCROOT/components/cam/cime_config/usermods_dirs/rcemip</value>
```

Comment:
See comment above about globbing for both MMF1 and MMF2.

Reply:
yea, I'll fix this too